components/Dashboard/Cards/DashboardCard.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Paper, Box, Typography } from "@material-ui/core";
+import { useTheme } from "@material-ui/core/styles";
+
+export default function DashboardCard(props) {
+  const theme = useTheme();
+  const { spaceName, description, variant } = props;
+  return (
+    <Box
+      style={{ background: variant==="primary" ? theme.palette.primary.mainGradient : theme.palette.secondary.mainGradient}}
+      className="flex flex-grow justify-between rounded-r-xl rounded-bl-xl "
+    >
+      <div class="h-32 w-80 grid grid-cols-3">
+        <div class="p-6">
+          {/* spaceholder */}
+          <Box class="h-20 w-20 bg-purple-200"></Box>
+        </div>
+        <div class="col-span-2 pt-2">
+          <Box className="text-left p-3">
+            <Typography color="text" variant="h5" align="left">
+              {spaceName}
+            </Typography>
+            <Typography variant="body1" className="text-sm pt-1">
+              {description}
+            </Typography>
+          </Box>
+        </div>
+      </div>
+    </Box>
+  );
+}
```

Comment:
missing new line

Reply:
make sure you have eslint on. It should've asked you to add PropTypes. It's not added here tho